### PR TITLE
*: add Vagrant guides

### PR DESCRIPTION
**Pull request by robszumski**

### CODE REVIEW

**Comment by philips** [comment]
@robszumski I think we should reorder this a bit as running all of the components can take some time: https://github.com/coreos/coreos-kubernetes/pull/36

I am cool with moving this into the Documentation dirs though.

**Comment by philips** [comment]
LGTM, I am happy to rebase #36 post merge.

File: single-node/README.md [hunk 1]
```diff
@@ -1,63 +1,7 @@
...
-172.17.4.99   kubernetes.io/hostname=172.17.4.99   Ready
-```
-
-## Step 3: Deploy an Application
-
-Now that you've got a working Kubernetes cluster with a functional CLI tool, you are free to deploy Kubernetes-ready applications.
-Start with a [multi-tier web application][guestbook] from the official Kubernetes documentation to visualize how the various Kubernetes components fit together.
-
-[guestbook]: http://kubernetes.io/v1.0/examples/guestbook-go/README.html
+Submit a PR to this repository. The source for the [docs is here](../../Documentation/kubernetes-on-vagrant-single.md)
```
Comment (bcwaldon) [inline single-node/README.md]: "The source for the docs is here" sounds weird...

"The source for this doc is here" or "The source of this documentation is here" ?

Comment (robszumski) [inline single-node/README.md]: ack

**Comment by bcwaldon** [comment]
LGTM

**Pull request merged**